pyani_plus/private_cli.py

```diff
@@ -348,28 +348,27 @@ def log_fastani(  # noqa: PLR0913
     if config.configuration_id is None:
         sys.exit("Error with configuration table?")
 
-    sim_errors = fragments - orthologous_matches
+    sim_errors = fragments - orthologous_matches  # proxy value, not bp
 
-    # Need to lookup query length to compute query_cover:
+    # Need to lookup query/subject length to estimate alignment length:
     query_md5 = file_md5sum(query_fasta)
     query = db_orm.add_genome(session, query_fasta, query_md5)
-    cov_query = float(orthologous_matches) / query.length
-
-    # Need to lookup subject length to compute subject_cover:
     subject_md5 = file_md5sum(subject_fasta)
     subject = db_orm.add_genome(session, subject_fasta, subject_md5)
-    cov_subject = float(orthologous_matches) / subject.length
+
+    estimated_cov_query = float(orthologous_matches) / fragments  # an approximation
+    estimated_aln_length = estimated_cov_query * min(query.length, subject.length)
 
     db_orm.add_comparison(
         session,
         configuration_id=config.configuration_id,
         query_hash=query_md5,
         subject_hash=subject_md5,
         identity=identity,
-        aln_length=orthologous_matches,
+        aln_length=estimated_aln_length,
```

Comment:
We don't have a real alignment length from fastANI.

One might argue by analogy with the current treatment of sim-errors (recording the number of left over fragments) that we record the number of orthologous matches?

Or, the estimate in bp using the estimated coverage fraction?